radicle-cli/src/git.rs

```diff
@@ -204,6 +248,15 @@ pub fn rad_remote(repo: &Repository) -> anyhow::Result<(git2::Remote, Id)> {
     }
 }
 
+pub fn rad_add_remote<'a>(
+    repo: &'a Repository,
+    name: &'a str,
+    url: &'a str,
```

Comment:
It should take the `Url` type instead of a string, guaranteeing it to create a `rad` remote and not any old URL remote